.github/workflows/cibuildwheel.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux2010
           CIBW_MANYLINUX_I686_IMAGE: manylinux2010
           CIBW_BEFORE_BUILD: >-
-            pip install certifi numpy==1.19.2 &&
+            pip install certifi numpy==1.20.3 &&
```

Comment:
I have a concern that this is higher than our minimum we tell the packaging system as I am not 100% sure that if we build a wheel with numpy 1.20.3 it will work with numpy 1.20.0.  I suspect it will be OK, but we should be sure before we merge.

Reply:
Yeah, I didn't really understand why it was 1.19.2 earlier and not, say, 1.19.0. (Anyway, since `oldest_supported_numpy` will be installed in the isolated, I *think* that is the one being used, so I'll try removing this completely and see what happens...)

Reply:
This pin is for one of the cases that `oldest-supported-numpy` does not cover. I _think_ it would be aarch64, but it's possibly fine now. You may be able to remove this special case and use `oldest-supported-numpy` everywhere, but should check the build carefully to confirm.

Reply:
I take it that the safe thing is to just keep this as is.